foundations/html_css/html-foundations/html-boilerplate.md

```diff
@@ -158,7 +158,7 @@ But it's still good to know how to write the boilerplate yourself in case you fi
 
 2.  Build some muscle memory by deleting the contents of the `index.html` file and trying to write out all the boilerplate again from memory. Don't worry if you have to peek at the lesson content the first few times if you get stuck. Just keep going until you can do it a couple of times from memory.
 
-3.  Run your boilerplate through an [HTML validator](https://validator.w3.org/). Validators ensure your markup is correct and are an excellent learning tool, as they provide feedback on syntax errors you may be making often and aren't aware of, such as missing closing tags and extra spaces in your HTML.
+3.  Run your boilerplate through an [HTML validator](https://www.freeformatter.com/html-validator.html). Validators ensure your markup is correct and are an excellent learning tool, as they provide feedback on syntax errors you may be making often and aren't aware of, such as missing closing tags and extra spaces in your HTML.
```

Comment:
If we could change it to "Run your boilerplate through the W3 [HTML validator](https://validator.w3.org/) or alternatively this [HTML validator](https://www.freeformatter.com/html-validator.html)". I would be happy to approve this PR.

Reply:
Updated, that was a good idea since the learners has more options and they can pick the another link if one doesn't work.